__tests__/integration-tests/routes/news.test.js

```diff
@@ -43,4 +43,17 @@ describe('/api/news/sources', () => {
     expect(response.body).toBeTruthy();
     expect(SUPPORTED_REGIONS).toEqual(responseRegions)
   });
+})
+
+describe('/api/news/sources/:sourceId', () => {
+  beforeEach(() => mockAxios.get('_')  );
+
+  it('should return a list of sources having same sourceId(i.e name)', async () => {
+    
+    const response = await request.get('/api/news/sources/thetimes');
+
+    expect(mockAxios.get).toHaveBeenCalledTimes(2);
+    expect(response.status).toBe(200);
+    expect(response.body).toBeTruthy();
```

Comment:
It would be awesome if you add another expectation to see if all the articles from the response body have the same source.
You can check out previous test suites.
We can merge shortly after that! :rocket:

Reply:
Okay!

Thank you for the review and I'll check it out asap!